modules.json

```diff
@@ -0,0 +1,358 @@
+  },
+
+  {
+    "repo_name": "puppetlabs-ntp",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "ntp"
+  },
+
+
+  {
+    "repo_name": "puppetlabs-package",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "package"
+  },
+
+  {
+    "repo_name": "puppetlabs-postgresql",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "postgresql"
+  },
+
+  {
+    "repo_name": "puppetlabs-powershell",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "powershell"
+  },
+
+  {
+    "repo_name": "puppetlabs-puppet_conf",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "puppet_conf"
+  },
+
+  {
+    "repo_name": "puppetlabs-reboot",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "reboot"
+  },
+
+  {
+    "repo_name": "puppetlabs-registry",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "registry"
+  },
+
+  {
+    "repo_name": "puppetlabs-resource",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "resource"
+  },
+
+  {
+    "repo_name": "puppetlabs-resource_api",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "resource_api"
+  },
+
+  {
+    "repo_name": "puppetlabs-rook",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "rook"
+  },
+
+  {
+    "repo_name": "puppetlabs-scheduled_task",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "scheduled_task"
+  },
+
+  {
+    "repo_name": "puppetlabs-service",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "service"
+  },
+
+  {
+    "repo_name": "puppetlabs-sqlserver",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "sqlserver"
+  },
+
+  {
+    "repo_name": "puppetlabs-stdlib",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "stlib"
+  },
+
+  {
+    "repo_name": "puppetlabs-tagmail",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "tagmail"
+  },
+
+  {
+    "repo_name": "puppetlabs-tomcat",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "tomcat"
+  },
+
+  {
+    "repo_name": "puppetlabs-translate",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "translate"
+  },
+
+  {
+    "repo_name": "puppetlabs-vcsrepo",
+    "github_namespace": "puppetlabs",
+    "forge_namespace": "puppetlabs",
+    "module_name": "vcsrepo"
+  },
+
+  {
+    "repo_name": "puppetlabs-websphere_application_server",
```

Comment:
duplicate entry - see L16. 

would be nice to have a formatter/checker for this file that keeps it in alphabetical order and checks for duplicates and missing keys.